packages/components/src/components/hds/application-state/footer.hbs

```diff
@@ -4,5 +4,11 @@
 }}
 
 <div class="hds-application-state__footer" ...attributes>
-  {{yield (hash Button=(component "hds/button") LinkStandalone=(component "hds/link/standalone"))}}
+  {{yield
+    (hash
+      Button=(component "hds/button")
+      LinkStandalone=(component "hds/link/standalone")
+      Dropdown=(component "hds/dropdown")
+    )
+  }}
```

Comment:
@zamoore Was this update meant to be done in this PR vs [this](https://github.com/hashicorp/design-system/pull/2274)?

Reply:
Wait, are we allowing to use a dropdown in the footer of the application state? 🤔

Reply:
Yes - l[ink to comment.](https://github.com/hashicorp/design-system/pull/2274/files#r1693318678)

Reply:
@majedelass see https://github.com/hashicorp/design-system/pull/2274/files#r1696874211

Reply:
Yes, let me move this over. 🤦